pytorch/Dockerfile

```diff
@@ -9,4 +9,4 @@ RUN apt-get update && apt-get install -y rsync && \
     pip install ruamel.yaml --ignore-installed
 
 # NERSC TensorBoard Jupyter helper
-COPY nersc-tensorboard-helper/nersc_tensorboard_helper.py /opt/conda/lib/python3.6/site-packages
+COPY nersc-tensorboard-helper/nersc_tensorboard_helper.py /opt/conda/lib/python3.8/site-packages
```

Comment:
maybe we can wildcard this or something, so we don't have to hardcode the python version in the path

Reply:
Yes, maybe we can do that, at least until it breaks one day.